specification/attestation/data-plane/Microsoft.Attestation/preview/2018-09-01-preview/attestation.json

```diff
@@ -0,0 +1,304 @@
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+    "paths": {
+      "/operations/update": {
+            "post": {
+                "tags": [
+                         "attestation",
+                         "policy"
+                        ],
+                "summary": "Accepts a new policy document and returns a JWT which expresses  used in preparation to set attestation policy.",
+                "operationId": "PrepareToSetPolicy",
+                "consumes": [ "text/plain" ],
+                "produces": [ "text/plain", "application/json" ],
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/AuthorizationParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/TeeKind"
+                    },
+                    {
+                        "name": "PolicyJws",
+                        "in": "body",
+                        "description": "JSON Web Signature (See RFC7515) expressing the new policy",
+                        "required": true,
+                        "x-ms-client-flatten": true,
+                        "schema": {
+                            "type": "string"
+                        }
+                    }
+                ],
+                "responses": {
+                  "200": {
+                    "description": "Success - Returns a JWT signed by the metadata signing key that contains the hash of the supplied policy to be set.",
+                    "schema": {
+                      "$ref": "#/definitions/AttestationPolicy"
+                    }
+                  },
+                    "400": {
+                        "description": "Bad request",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    },
+                    "401": {
+                        "description": "Request is unauthorized"
+                    }
+                },
+                "x-ms-examples": {
+                    "UpdatePolicy": {         
+                      "$ref": "./examples/UpdatePolicy.json"         
+                    }
+                }
+            }
+        },
+        "/operations/policy/current": {
+            "get": {
+                "tags": [
+                         "attestation",
+                         "policy"
+                        ],
+                "summary": "Retrieves the current policy for a given kind of TEE.",
+                "operationId": "GetPolicy",
```

Comment:
operationId naming convention: Resource_Action

Reply:
make the following changes:
PrepareToSetPolicy->Policy_PrepareToSet
SetPolicy->Policy_Set
GetPolicy->Policy_Get
DeletePolicy->Policy_Delete